site/en/external/migration.md

```diff
@@ -32,6 +32,19 @@ your workspace root, maybe with comments like:
     # See MODULE.bazel for external dependencies setup.
     ```
 
+### Enable Blzmod in your bazelrc {:#enable-bzldmod}
+
+`.bazelrc` lets you set flags that apply every time your run Bazel.  To enable
+Bzlmod we use the `--enable_bzlmod` flag, and apply it to the `common` command so
```

Comment:
"To enable Bzlmod, use the" would be better. We tend to avoid using "we" (hah) in docs.

Reply:
All fixed, thanks for the correction.